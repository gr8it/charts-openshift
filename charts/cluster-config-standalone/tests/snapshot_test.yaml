suite: Snapshot test
templates:
  - templates/*
values:
  - ../values.example.yaml
release:
  name: cluster-config-standalone
  namespace: openshift-config

tests:
  - it: manifests should match snapshot
    asserts:
      - matchSnapshot: {}
