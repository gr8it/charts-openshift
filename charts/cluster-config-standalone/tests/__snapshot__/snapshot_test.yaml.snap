manifests should match snapshot:
  1: |
    apiVersion: v1
    data:
      quay.apps.example.com..443: |
        ---BEGIN CERTIFICATE---
        MIIDdzCCAl+gAwIBAgIEb1pQszANBg
        ---END CERTIFICATE---
    kind: ConfigMap
    metadata:
      labels:
        app.kubernetes.io/instance: cluster-config-standalone
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: cluster-config-standalone
        helm.sh/chart: cluster-config-standalone-1.0.0
      name: registry-ca-bundle
      namespace: openshift-config
  2: |
    apiVersion: external-secrets.io/v1beta1
    kind: ExternalSecret
    metadata:
      labels:
        app.kubernetes.io/instance: cluster-config-standalone
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: cluster-config-standalone
        helm.sh/chart: cluster-config-standalone-1.0.0
      name: pull-secret
      namespace: openshift-config
    spec:
      data:
        - remoteRef:
            key: apc-platform/d/cluster-config
            property: pull-secret
          secretKey: pullsecret
      refreshInterval: 1h
      secretStoreRef:
        kind: ClusterSecretStore
        name: vault-issuer
      target:
        creationPolicy: Owner
        name: pull-secret
        template:
          data:
            .dockerconfigjson: '{{ .pullsecret | toString }}'
          type: kubernetes.io/dockerconfigjson
  3: |
    apiVersion: config.openshift.io/v1
    kind: Image
    metadata:
      annotations:
        argocd.argoproj.io/sync-options: ServerSideApply=true
      name: cluster
    spec:
      additionalTrustedCA:
        name: registry-ca-bundle
  4: |
    apiVersion: config.openshift.io/v1
    kind: ImageDigestMirrorSet
    metadata:
      labels:
        app.kubernetes.io/instance: cluster-config-standalone
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: cluster-config-standalone
        helm.sh/chart: cluster-config-standalone-1.0.0
      name: quay.apps.example.com
    spec:
      imageDigestMirrors:
        - mirrors:
            - quay.apps.example.com/registry.redhat.io
          source: registry.redhat.io
        - mirrors:
            - quay.apps.example.com/registry.connect.redhat.com
          source: registry.connect.redhat.com
        - mirrors:
            - quay.apps.example.com/docker.io
          source: docker.io
        - mirrors:
            - quay.apps.example.com/quay.io
          source: quay.io
        - mirrors:
            - quay.apps.example.com/ghcr.io
          source: ghcr.io
  5: |
    apiVersion: config.openshift.io/v1
    kind: ImageTagMirrorSet
    metadata:
      labels:
        app.kubernetes.io/instance: cluster-config-standalone
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: cluster-config-standalone
        helm.sh/chart: cluster-config-standalone-1.0.0
      name: quay.apps.example.com
    spec:
      imageTagMirrors:
        - mirrors:
            - quay.apps.example.com/registry.redhat.io
          source: registry.redhat.io
        - mirrors:
            - quay.apps.example.com/registry.connect.redhat.com
          source: registry.connect.redhat.com
        - mirrors:
            - quay.apps.example.com/docker.io
          source: docker.io
        - mirrors:
            - quay.apps.example.com/quay.io
          source: quay.io
        - mirrors:
            - quay.apps.example.com/ghcr.io
          source: ghcr.io
